eva_sub_cli/validators/validator.py

```diff
@@ -86,17 +89,33 @@ def validate_and_report(self):
         self.report()
 
     def validate(self):
+        self.set_up_output_dir()
         self.verify_files_present()
         self._validate()
         self._collect_validation_workflow_results()
 
     def report(self):
-        self.create_reports()
+        report_path = self.create_reports()
         self.update_config_with_validation_result()
+        self.clean_up_output_dir(report_path)
 
     def _validate(self):
         raise NotImplementedError
 
+    def set_up_output_dir(self):
+        if os.path.exists(self.output_dir):
+            shutil.rmtree(self.output_dir)
+        os.makedirs(self.output_dir, exist_ok=True)
+
+    def clean_up_output_dir(self, report_path):
+        # Move everything that's not report.html into a subdir
+        subdir = os.path.join(self.output_dir, 'other_validations')
+        os.mkdir(subdir)
+        for file_name in os.listdir(self.output_dir):
+            file_path = os.path.join(self.output_dir, file_name)
+            if os.path.isfile(file_path) and file_path != report_path:
+                os.rename(file_path, os.path.join(subdir, file_name))
```

Comment:
This will only work for the first time we rerunning a validation. 
if the destination exists `rename` will fail.

Reply:
I think, because we remove/rename the validation output directory every time, the destination should never exist already...